cg_hermes/config/rnafusion.py

```diff
@@ -90,6 +90,36 @@
         "tags": ["samplesheet-valid"],
         "used_by": ["cg"],
     },
+    frozenset({"multiqc-fastp, multiqc"}): {
+        "is_mandatory": True,
+        "tags": ["qc-metrics", "multiqc", "fastp"],
+        "used_by": ["storage", "janus"],
```

Comment:
`storage` has been deprecated
```suggestion
        "used_by": ["janus"],
```